### DISCUSSION TITLE
owstaxidmetadatalltagsinfo

### DISCUSSION THREAD

**Discussion by mahakjainau** [microsoft-search/pnp-modern-search#2058] [General]
![image](https://user-images.githubusercontent.com/13897223/166412684-03a57bd6-c7fb-4dcf-87ea-c5e57f568df5.png)

I would like to display the metadata info under the file that has been searched in a readable format. Could someone please help?
This information goes away when I untick "owstaxidmetadatalltagsinfo" from Search Results > selected properties.

**Comment by FranckyC** [comment]
Hi @mahakjainau, you need to enable the localization in the SharePoint Search options and then, in the slots configuration, use the `Autoowstaxidmetadatalltagsinfo` field for slot `Tags` (type the field name and press 'Enter' to validate the value).

![image](https://user-images.githubusercontent.com/5659122/166678312-0eb010ba-71cc-4dc6-8e4a-28f6c64607e4.png)

![image](https://user-images.githubusercontent.com/5659122/166678376-651b0629-9eb7-419c-bc60-8157dcff8c3e.png)

See https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/#sharepoint-search

**Comment by mahakj-mojosoup** [comment]
Hi Francky,

Thanks for your reply. Appreciate it. This has worked.
Is there a way I can format the way the tags are getting displayed? What I mean is - I just want to add some spacing to the Tags so they are clearer to read.

Cheers
Mahak

**Reply by FranckyC** [reply]
Glad it worked! You have full control over the HTML/Handlebars template so you can put some CSS to format the tags the way you want https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#custom-css-styles

**Reply by linahin** [reply]
@mahakjainau were you able to figure out how to have a space between the tags? I'm having the same behavior. Thanks
